.github/workflows/build-push-supernova-image.yaml

```diff
@@ -0,0 +1,108 @@
+# Run it locally with act
+#  1. Install act:
+#     `brew install act`
+#  2. Create a .secret file with the following content:
+#     `GITHUB_TOKEN=your_github_token`
+#  WORKFLOW_DISPATCH
+#     `act workflow_dispatch  --container-architecture linux/amd64 -P default=catthehacker/ubuntu:act-latest -W .github/workflows/build-push-supernova-image.yaml`
+
+name: Supernova UI Image
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+    paths:
+      - alerts/ui/docker/VERSION
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: "juno-app-supernova"
+  PACKAGE_PATH: "alerts/ui"
+
+jobs:
+  build-and-push:
+    name: Build&Push
+    runs-on: [default]
+
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # This action enables you to SIGN and VERIFY container images using cosign
+      # https://github.com/sigstore/cosign-installer
+      - name: Install cosign
```

Comment:
Why `cosign` is installed when not using it and signing the image like here: 
https://github.com/cloudoperators/greenhouse/blob/cd0e226dc8e272106d15f05b60d37667a2e7dc0a/.github/workflows/docker-build.yaml#L128-L136

Reply:
This is planed for the second iteration. The build and push step is for now deactivated. We want just see the action in action :)